mergify_engine/actions/merge/action.py

```diff
@@ -38,6 +38,16 @@
 MARKDOWN_TITLE_RE = re.compile(r"^#+ ", re.I)
 MARKDOWN_COMMIT_MESSAGE_RE = re.compile(r"^#+ Commit Message ?:?\s*$", re.I)
 
+PRIORITY_ALIASES = {
+    "low": 0,
```

Comment:
I'm not sure I'd put 0. You may have people who wants to override low at somepoint with something lower.

I'd go with interval that are more like
`low`: 1000
`medium`: 2000
`high`: 3000


which then can make it easy to read prio `1xxx` are the low ones, `0xxx` are the lowest ones, `2xxx` the medium ones, etc. for people hacking around.